drop-transfer/src/error.rs

```diff
@@ -111,14 +112,14 @@ impl From<&Error> for i32 {
 }
 
 pub trait ResultExt {
-    fn to_status(&self) -> Result<(), i32>;
+    fn to_status(&self) -> i32;
 }
 
 impl<T> ResultExt for super::Result<T> {
-    fn to_status(&self) -> Result<(), i32> {
+    fn to_status(&self) -> i32 {
         match self {
-            Ok(_) => Ok(()),
-            Err(err) => Err(u32::from(err) as _),
+            Ok(_) => MOOSE_STATUS_SUCCESS,
+            Err(err) => u32::from(err) as _,
```

Comment:
Since it's now explicitly using moose status codes, maybe it's worth renaming it to something like `to_moose_status()`? So that it's not used by accident in some other context

Reply:
good point, will do